src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -70,12 +70,21 @@ private void changeTabs(IXtraTabPage page, IXtraTabPage prevPage)
             prevPage.Appearance.Header.Font = Fonts.NonSelectedTabHeaderFont;
       }
 
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         layoutItemSelectRange.AdjustLongButtonSize();
+         layoutItemImportAll.AdjustButtonSize();
+         layoutItemImport.AdjustButtonSize();
+         btnSelectRange.InitWithImage(ApplicationIcons.PreviewOriginData, Captions.Importer.PreviewExcelData);
+         resetImportButtonText();
```

Comment:
I had to remove all buttons and add them again because for some reason, their default behavior had been changed and thus our standard method were not working. Always a joy to work with the importer